docs/examples/repeated_regions.py

```diff
@@ -0,0 +1,19 @@
+from pypom import Page, Region
+from selenium.webdriver.common.by import By
+
+
+class Results(Page):
+    _result_locator = (By.CLASS_NAME, 'result')
+
+    @property
+    def results(self):
+        results_element = self.find_element(*self._result_locator)
+        items = results_element.find_elements(*self.Result._root_locator)
+        return [self.Result(self, el) for el in items]
+
+    class Result(Region):
+        _root_locator = (By.CLASS_NAME, 'item')
```

Comment:
Again, `_root_locator` is more for regions that are unique on the page. By defining it for repeated regions I would expect issues to occur.